baldrick/plugins/github_pull_requests.py

```diff
@@ -138,7 +138,7 @@ def process_pull_request(repository, number, installation, action,
     results = {}
     for function, actions in PULL_REQUEST_CHECKS.items():
         if actions is None or action in actions:
-            result = function(pr_handler, repo_handler)
+            result = function(pr_handler, repo_handler, payload)
```

Comment:
Does this need to break backward compat? Can this be an optional keyword as above?

Reply:
this isn't a function sig, we are calling the registered callback functions. The only way to pass this information is to always pass it. We *could* do some crazy signature introspection stuff but...

Reply:
Yeah, I see your point... 🤔